### PR TITLE
[CI:DOCS] Cirrus: Fix win installer task

### PR DESCRIPTION
**Pull request by cevich**
ref: https://github.com/containers/podman/pull/16167

According to Cirrus-support, defining and using `$CIRRUS_DEFAULT_WORK`
is responsible for causing a strange race condition on windows.  This is
impacting the agent's ability to clone with errors like:

```
Using built-in Git...
Cloning refs/heads/main...
Failed to clone: read ${CIRRUS_DEFAULT_WORK}\.git\HEAD: The process
cannot access the file because another process has locked a portion of
the file.!
```

Fix this by hard-coding a work directory for this windows task.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by cevich** [comment]
@n1hility PTAL when you get a chance.  I *think* this is the right thing to do, but am certainly uncertain :grin:

**Comment by n1hility** [comment]
@cevich im not sure if this should be in a local tmp, let me check what the default normally is

**Review by n1hility — CHANGES_REQUESTED** [review]
A root location is probably fine, but I posted a suggestion above to put it under the users temp dir which matches what they do by default.

**Comment by cevich** [comment]
Thanks Jason.

This should be ready to merge assuming CI is happy.

**Comment by cevich** [comment]
@containers/podman-maintainers PTAL - this is affecting all of CI.

**Comment by baude** [comment]
LGTM

**Pull request merged**
